Framework/API/src/ScriptRepository.cpp

```diff
@@ -45,9 +19,7 @@ ScriptRepoException::ScriptRepoException(const std::string &info, const std::str
   } else {
     _file_path = file;
     if (line > 0) {
-      char aux[10];
-      snprintf(aux, 10, ":%d", line);
-      _file_path.append(aux);
+      _file_path.append(std::to_string(line));
```

Comment:
I think we are missing a `:` from the string:
```suggestion
      _file_path.append(": ").append(std::to_string(line));
```